test/ImportJSONStorageGraph.scala

```diff
@@ -52,3 +52,11 @@ object ImportJSONStorageNoBucketsGraph {
     graph.io( IoCore.graphson() ).reader().create().readGraph( inputStream, graph )
   }
 }
+
+object ImportJSONStorageNoFilesGraph {
+
+  def populateGraph( graph: Graph ): Unit = {
```

Comment:
I think at this point it's time to factor code ;-)

Reply:
Yup, almost done :)